src/cpp/rtps/participant/RTPSParticipantImpl.h

```diff
@@ -126,7 +126,24 @@ class RTPSParticipantImpl
         * @param part
         * @param plisten
         */
-    RTPSParticipantImpl(const RTPSParticipantAttributes &param, const GuidPrefix_t& guidP, RTPSParticipant* part,
+    RTPSParticipantImpl(
+        const RTPSParticipantAttributes &param,
+        const GuidPrefix_t& guidP,
+        RTPSParticipant* part,
+        RTPSParticipantListener* plisten = nullptr);
+
+    /**
+        * @param param
+        * @param guidP
+        * @param persistence_guid
+        * @param part
+        * @param plisten
+        */
+    RTPSParticipantImpl(
```

Comment:
Wrong indentation on comments and parameters

Reply:
ok